packages/core/upload/admin/src/hooks/useAssets.js

```diff
@@ -0,0 +1,50 @@
+import { useEffect } from 'react';
+import { useQuery } from 'react-query';
+import { useNotifyAT } from '@strapi/parts/LiveRegions';
+import { useQuery as useURLQuery, useNotification } from '@strapi/helper-plugin';
+import { useIntl } from 'react-intl';
+import { axiosInstance, getRequestUrl, generateStringFromParams } from '../utils';
+import useSelectTimestamps from './useSelectTimestamps';
+
+export const useAssets = ({ skipWhen }) => {
+  const { formatMessage } = useIntl();
+  const toggleNotification = useNotification();
+  const [, updated_at] = useSelectTimestamps();
+  const { notifyStatus } = useNotifyAT();
+  const query = useURLQuery();
+  const dataRequestURL = getRequestUrl('files');
+  const params = generateStringFromParams(query);
```

Comment:
You can delete this utility as we will use `qs` and the values returned by the useQueryParams hook.